### ISSUE TITLE
JAWS does not read keyboard shortcuts correctly

### ISSUE THREAD

**Issue by CKFinderBOT** [ckfinder/ckfinder#209]
JAWS 16.0.1722 + FF 40.0.3

Open the Keyboard Shortcuts dialog. Note that the content inside `<span class="ckf-shortcuts-reader-only">` is ignored. Instead JAWS just read the arrow keys in its own way (e.g. *upward zero* or sth similar) 

The following content might be handy: http://terrillthompson.com/tests/hiddencontent.html

Btw. this article: http://john.foliot.ca/aria-hidden/ suggests using `aria-hidden` with `role="presentation"` to hide content from screen readers.

**Comment by wwalc** [comment]
Fixed. The updated code will be publicly available in the upcoming release.
